tests/functional/send_sms.js

```diff
@@ -113,7 +113,7 @@
              country: 'RO'
            }
          }))
-         .then(testElementValueEquals(selectors.SMS_SEND.PHONE_NUMBER, '+407'))
+         .then(testElementValueEquals(selectors.SMS_SEND.PHONE_NUMBER, '+40'))
```

Comment:
Slight modification from previous behavior. To generalize the formatting logic, only pre-fill the country code for RO.